### PR TITLE
Added Assistive Technology Compatibility section

### PR DESCRIPTION
**Pull request by WestonThayer**
The Microsoft Edge accessibility docs provide an excellent introduction to building for web accessibility. This PR adds information and resources for assistive technology compatibility testing, which is a great way to validate that the code you're writing works with ATs like screen readers.

I structured it similarly to the [Emulate and test other browsers](https://docs.microsoft.com/en-us/microsoft-edge/devtools-guide-chromium/device-mode/testing-other-browsers) doc.

### CODE REVIEW

**Comment by codepo8** [comment]
This looks great to me. The only feedback is that it is a bit wordy and we could consider to add a section on keyboard accessibility. As in - even if you can't test with a screen reader, a good first step is to ensure that you can navigate your products with a keyboard. If that isn't possible, a screenreader user will never be able to access your content.

**Comment by WestonThayer** [comment]
Thanks for the review @codepo8, great idea. I've added a section on keyboard accessibility and tried to cut out some of the more verbose paragraphs. Feedback welcome!

**Comment by WestonThayer** [comment]
@codepo8 are you comfortable with approving?

**Comment by codepo8** [comment]
Yes, this is approved #sign_off

**Review by jm-trd-ms — APPROVED** [review]

**Comment by jm-trd-ms** [comment]
#sign-off and merged.

**Comment by jm-trd-ms** [comment]
@WestonThayer 
Thank you for contributing to our content.  Your contribution had resulted in changes to our documentation.
